src/Site/Site.php

```diff
@@ -196,17 +196,26 @@ public static function setLastSegment($segments)
     }
 
     /**
-     * generate salts for files.
+     * Generate salts.
      *
-     * @param (int) $length Length of salts.
+     * @param (int)  $length  Length of salts.
+     * @param (bool) $special Should special chars include or not.
      *
      * @since 1.0.0
      *
      * @return string
      */
-    public static function salts($length)
+    public static function salts(int $length, $special = false)
```

Comment:
It should be `public static function salts(int $length, bool $special = false)`.